dashboard/src/pages/TreeDetails/TreeDetails.tsx

```diff
@@ -28,6 +36,46 @@ export interface ITreeDetails {
   builds: AccordionItemBuilds[];
 }
 
+const TreeHeader = (): JSX.Element => {
+  return (
+    <DumbBaseTable>
+      <DumbTableHeader>
+        <TableHead>
+          <FormattedMessage id="global.tree" />
+        </TableHead>
+        <TableHead>
+          <FormattedMessage id="global.branch" />
+        </TableHead>
+        <TableHead>
+          <FormattedMessage id="treeDetails.commitOrTag" />
+        </TableHead>
+        <TableHead>
+          <FormattedMessage id="global.url" />
+        </TableHead>
+        <TableHead>
+          <FormattedMessage id="global.estimate" />
+        </TableHead>
+        <TableHead>
+          <FormattedMessage id="global.status" />
+        </TableHead>
```

Comment:
i'm not sure we are going to show this, we don't have the estimate time information or the tree status